Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "hs-builder-core"
-version = "0.1.1"
+name = "hotshot-builder-core"
```

Comment:
Name change is good, but we should rename the github repo at some point soon. Like, as soon as we get this published to crates.io

Reply:
Yeah, We can change the name after this PR merge. https://github.com/EspressoSystems/hs-builder-core/pull/68/commits/832462931d7160e8b11e661e9859098fd5f20cc3